readthedocs/core/resolver.py

```diff
@@ -214,6 +214,58 @@ def resolve(
         )
         return urlunparse((protocol, domain, path, '', query_params, ''))
 
+    def _get_root_path(self, project):
+        """
+        Returns the root path for a project.
```

Comment:
Is root path a term we've used before? We should document the exact terms we call these things somewhere... but not sure the best place.

Reply:
Don't think we have used that before, I know in the old implementation of proxito we refer to these as `/` redirect, because we redirect them to their default version, but outside that nothing.

Reply:
we could use `proxito/README.rst`